tests/ci/azure_pipeline_test/dsvm_linux_template.yml

```diff
@@ -21,51 +21,103 @@ parameters:
 - name: pytest_params
   type: string
   default: ""
+- name: install
+  type: string
+  default: ""
+- name: package
+  type: string
+  default: ""
 
 jobs:
-- job: test
+- job: "${{ parameters.conda_env }}" # Job name can't have spaces
   displayName: "${{ parameters.task_name }}"
   timeoutInMinutes: ${{ parameters.timeout }}
   pool:
     name: $(Agent_Pool)
 
   steps:
+
   - script: |
       echo "##vso[task.prependpath]/data/anaconda/bin"
-      conda create -fyn ${{ parameters.conda_env }} ${{ parameters.conda_opts }}
 
-      # install reco-utils and needed dependencies
+      # Make sure there is no conda environment
+      conda env remove -n ${{ parameters.conda_env }} || exit -1
+
+      # Create a clean conda environment
+      conda create -fyn ${{ parameters.conda_env }} ${{ parameters.conda_opts }}
       eval "$(conda shell.bash hook)"
       conda activate ${{ parameters.conda_env }}
-      pip install .${{ parameters.pip_opts }}
-      pip install -r dev-requirements.txt
+
+      # install reco-utils and needed dependencies
+      if [ "${{ parameters.install }}" = "release" ]; then
+        echo "   --- SET UP PARAMETERS ---"
+        ARCH=$(uname -m)
+        if [[ $ARCH == "x86_64" ]]; then
+            PLATFORM="manylinux1_x86_64"
+        else
+            PLATFORM="manylinux2014_$ARCH"
+        fi
+        RELEASE_VERSION=$(awk '/__version__ = / {print $3}' reco_utils/__init__.py | sed 's/"//g')
+
+        echo "   --- BUILDING PACKAGE ---"
+        rm -rf dist || exit -1
+
+        python setup.py sdist bdist_wheel --plat-name=$PLATFORM || exit -1
+
+        echo "   --- INSTALLING WHEEL ---"
+        pip install dist/ms_recommenders-$RELEASE_VERSION-py3-none-$PLATFORM.whl${{ parameters.pip_opts }} || exit -1
+      else
+        echo "   --- INSTALLING LATEST CODE ---"
+        pip install .${{ parameters.pip_opts }} || exit -1
+      fi
+      
+      echo "   --- LIST INSTALLED DEPENDENCIES ---"
+      pip list
 
       # clean up any previous test results        
       rm -rf reports
-    displayName: Create Conda Env
-  
+    displayName: 'Install Dependencies'
+
   - ${{ each test in parameters.test_types }}:
     - script: |
         eval "$(conda shell.bash hook)"
+
         conda activate ${{ parameters.conda_env }}
-        export PYSPARK_PYTHON=`which python`
-        export PYSPARK_DRIVER_PYTHON=`which python`
+        pip install pytest>=3.6.4 || exit -1
```

Comment:
curious why this was moved out of dev-requirements.txt and into the pipeline?

Reply:
same answer here: https://github.com/microsoft/recommenders/pull/1427#discussion_r649911784, closing this one so we follow the conversation in the other message